lab-31-chris/app/component/gallery/gallery-item/gallery-item.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('./_gallery-item.scss');
+
+module.exports = {
+  template: require('./gallery-item.html'),
+  controllerAs: 'galleryItemCtrl',
+  controller: ['$log', '$rootScope', 'galleryService', function($log, $rootScope, galleryService) {
+    $log.debug('Gallery Item Controller');
+
+    this.showEditGallery = false;
+
+  }],
+  bindings: {
+    gallery: '<',
```

Comment:
how do you think you could pass methods through bindings to reduce clutter of controllers?